lib/rubocop/cop/chef/modernize/chef_14_resources.rb

```diff
@@ -32,10 +32,10 @@ module ChefModernize
         #   depends 'sysctl'
         #
         class UnnecessaryDependsChef14 < Cop
-          MSG = "Don't depend on cookbooks made obsolete by Chef 14".freeze
+          MSG = "Don't depend on cookbooks made obsolete by Chef Infra Client 14+".freeze
```

Comment:
Do we wanna match the message above and also defined in the `cookstyle.yml`?
```suggestion
          MSG = "Don't depend on cookbooks made obsolete by Chef Infra Client 14+. These community cookbooks contain resources that are now included in Chef Infra Client itself.".freeze
```